config/manager/manager.yaml

```diff
@@ -29,6 +29,8 @@ spec:
         - /manager
         args:
         - --leader-elect
+        - --metrics-bind-address=:8080
```

Comment:
Maybe this line should be in `config/default/manager_metrics_patch.yaml`, because they are all metrics about

Reply:
sorry for that